tools.go

```diff
@@ -17,7 +17,7 @@
 package postgres
 
 import (
-	_ "github.com/conduitio/conduit-commons/paramgen"
+	_ "github.com/conduitio/conduit-connector-sdk/specgen"
```

Comment:
We'll have to change this once https://github.com/ConduitIO/conduit-connector-sdk/pull/246 lands.